src/postgres-util/src/schemas.rs

```diff
@@ -49,7 +49,6 @@ pub async fn publication_info(
     ssh_tunnel_manager: &SshTunnelManager,
     config: &Config,
     publication: &str,
-    oid_filter: Option<u32>,
```

Comment:
remove the docstring text above about `oid_filter`

Reply:
good catch, ty